static/bidder-params/adtelligent.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Adtelligent Adapter Params",
+  "description": "A schema which validates params accepted by the Adtelligent adapter",
+
+  "type": "object",
+  "properties": {
+    "placementId": {
+      "type": "integer",
+      "description": "An ID which identifies this placement of the impression"
+    },
+    "siteId": {
+      "type": "integer",
+      "description": "An ID which identifies the site selling the impression"
+    },
+    "sourceId": {
+      "type": "integer",
+      "description": "An ID which identifies the channel"
+    },
+    "bidFloor": {
+      "type": "number",
+      "description": "BidFloor, US Dollars"
```

Comment:
Yes, it's a min value and this field is not mandatory. We require only sourceId (aid attribute). Nothing more. So publishers are able to skip this attribute if they don't have any requirements to the bid.Price.

Reply:
Right. Sorry... what I meant was: It seems to me like a `bidFloor < 0` would be bad input, because it doesn't make much sense.

If you define a min value of "0" here,  pubs will get an error message about it.